.eslintrc

```diff
@@ -74,7 +74,6 @@
     "no-invalid-regexp": 2,          // http://eslint.org/docs/rules/no-invalid-regexp
     "no-irregular-whitespace": 2,    // http://eslint.org/docs/rules/no-irregular-whitespace
     "no-obj-calls": 2,               // http://eslint.org/docs/rules/no-obj-calls
-    "no-reserved-keys": 2,           // http://eslint.org/docs/rules/no-reserved-keys
```

Comment:
Would be replaced by [`quote-props`](http://eslint.org/docs/rules/quote-props), which would cause all inline objects to need quotes around the properties. I assumed that that wouldn't be wanted, because it would make this PR a lot bigger